thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/lib/FunctionReplayRunnable.java

```diff
@@ -176,14 +188,16 @@ public void run() {
 
     // clean up and kill itself
     if(selfKill) {
-      onboardResource.deleteExistingAnomalies(Long.toString(clonedFunctionId), replayStart.getMillis(),
+      onboardResource.deleteExistingAnomalies(Long.toString(clonedFunctionId), 0,
           replayEnd.getMillis());
       anomalyFunctionDAO.deleteById(clonedFunctionId);
```

Comment:
If is selfKipp, clonedFunctionId is also being deleted. What about the clonedFunctionId contained in Runnable?